src/vcSettings.cpp

```diff
@@ -92,6 +92,14 @@ bool SDL_filewrite(const char* filename, const char *pStr, size_t bytes)
   return success;
 }
 
+void vcSettings_Cleanup_CustomClassificationColorLabels(vcSettings *pSettings)
+{
+  size_t lenth = udLengthOf(pSettings->visualization.customClassificationColorLabels);
+  for (size_t i = 0; i < lenth; ++i)
```

Comment:
This is a multiline `for` so requires braces (there is another later as well)

Reply:
Fixed.